pygmt/tests/test_datasets_load_remote_datasets.py

```diff
@@ -35,8 +33,9 @@ def test_load_remote_dataset_benchmark_with_region():
     assert data.gmt.registration == 0
     assert data.shape == (11, 21)
     # The cpt attribute was added since GMT 6.4.0
-    if Version(__gmt_version__) >= Version("6.4.0"):
-        assert data.attrs["cpt"] == "@earth_age.cpt"
+    # Can't access the cpt attribute using virtual files
```

Comment:
Ok, so auto colormap won't work after this PR is merged? We'll need to remember to document this in the PyGMT v0.12.0 changelog.

Reply:
Yes, it will break. We may take different actions for input netCDF files or xarray.DataArray object. For xarray.DataArray, we can check whether it has the `cpt` attribute (makes more sense in the `gmt` accessor, xref #499); For netCDF files, we need to read the file header (header only so it should be quick) and check the `cpt` attribute. If given, then we add the same attribute to the output xarray.DataArray/netCDF. But we need to be careful, for example, `grdedit -D+c` can change the CPT, so the output grid can have different grid for the input grid.

For GMT remote datasets, we probably can add a CPT attribue to the `GMTRemoteDataset` class (https://github.com/GenericMappingTools/pygmt/blob/70d666e38d6bfac4132fc66d3c7adbeb57dc9588/pygmt/datasets/load_remote_dataset.py#L67).

Reply:
Ok, let's keep track about this `.cpt` attribute idea at https://github.com/GenericMappingTools/pygmt/issues/499#issuecomment-2058129481.